.github/actions/spell-check/expect.txt

```diff
@@ -1866,4 +1865,7 @@ MOZPL
 
 # FALSE POSITIVES
 
-declatory # NOTICE.MD > MOZILLA PUBLIC LICENSE 
\ No newline at end of file
+## NOTICE.MD > MOZILLA PUBLIC LICENSE v1.1
+declatory
+## "PackagemanagerWrapper.cs" should be "PackageManagerWrapper.cs"
+Packagemanager
```

Comment:
I'd suggest using `patterns.txt` for these -- you can use longer more precise patterns.

Reply:
I was thinking of simply renaming the file. Just wanna double-check the impact of that with people who know more about these files.

Reply:
The comment also applies to the `declatory` license item

Reply:
Source text needs to be updated. That'll resolve the issue.

Reply:
I think these should be added alphabetically. They'll just be moved around when someone runs the script to automatically try to update the file.